cdci_data_analysis/analysis/instrument.py

```diff
@@ -273,7 +273,7 @@ def check_instrument_query_role(self, query_obj, product_type, roles, par_dic):
     def get_html_draw(self, prod_name, image,image_header,catalog=None,**kwargs):
         return self.get_query_by_name(prod_name).get_html_draw( image,image_header,catalog=catalog,**kwargs)
 
-    def get_par_by_name(self,par_name):
+    def get_par_by_name(self,par_name, validate=False):
```

Comment:
not sure if this is needed really

Reply:
Currently is not doing anything so it is actually not needed, but we might want to implement it in the future

Reply:
So it's good to have it